nativeauthenticator/templates/change-password-admin.html

```diff
@@ -0,0 +1,58 @@
+{% extends "page.html" %}
+
+{% block script %}
+{{ super() }}
+<script type="text/javascript">
+    document.addEventListener('DOMContentLoaded', function() {
+        let button = document.getElementById('eye');
+        button.addEventListener("click", function(e) {
+            let npwd = document.getElementById("password_input");
+            let cpwd = document.getElementById("password_confirm");
+            if (npwd.getAttribute("type") === "password") {
+                npwd.setAttribute("type", "text");
+                cpwd.setAttribute("type", "text");
+                button.textContent = "🔑";
+            } else {
+                npwd.setAttribute("type", "password");
+                cpwd.setAttribute("type", "password");
+                button.textContent = "👁";
+            }
+        });
+    });
+</script>
+{% endblock script %}
+
+{% block main %}
+<div class="container">
+    <form action="{{post_url}}" method="post" role="form">
+        <h1>
+            Change password for {{user_name}}
+        </h1>
+        
+        <p>Please enter the new password you want to set for {{user_name}}.</p>
+
+        <div class="form-group">     
+            <label for="password_input">New password:</label>
+            <div class="input-group col-xs-6">
+                <input id="password_input" type="password" name="new_password" val="" autocapitalize="off" autocorrect="off" class="form-control" />
+                <span class="input-group-addon">
+                    <button id="eye" type="button" style="border:0;">👁</button>
+                </span>
+            </div>
+            <p></p>
+            
+            <label for="password_input">Confirm new password:</label>
+            <div class="input-group col-xs-6">
+                <input id="password_confirm" type="password" name="con_password" val="" autocapitalize="off" autocorrect="off" class="form-control" />
```

Comment:
I suggest we stick to including `_input` on all our input boxes, I've grown very appreciative of verbose variable names.

Perhaps `password_conf_input` or similar?

Reply:
Again, as above, `*_conf_*` for "confirmation", when it is so often used for "configuration" is not helpful, I think.
I went with `"old_password_input"`, `"new_password_confirmation_input"` and `"new_password_confirmation_input"`.